### PR TITLE
Strip rename change

### PR DESCRIPTION
**Pull request by clairecporter**

### CODE REVIEW

File: lib/dem.py [hunk 1]
```diff
@@ -66,7 +66,8 @@
                                     (-(?P<subtile>\d{2}))?
                                     _meta.txt\Z""", re.I | re.X)
 
-setsm_strip_pattern = re.compile("""(?P<pairname>
+setsm_strip_pattern = re.compile("""((?P<relversion>s2s[\d/]+)_)?
```
Comment (ehusby) [inline lib/dem.py:69]: I'd add the 'SETSM' part of the prefix here for completeness, unless you think it's better to keep it out with reasoning that this regex could be used to match a strip file that's created with a different algorithm (such as ASP).... but this is `index_setsm.py` after all and the variable is named `setsm_strip_pattern`

**Comment by ehusby** [comment]
Just one comment, otherwise looks like good to me!
Have you tested on some test renamed strip files?

File: lib/dem.py (hunk 1 above)
Comment (clairecporter) [reply lib/dem.py:69]: Sure, we can add the SETSM prefix optionally.  I believe the pattern is always used as re.search, so it is not functionally different, but I like your suggestion for completeness and safety.

**Comment by clairecporter** [comment]
Yes, I tested on the rema 01 set, but I will also wrap some of these into a test in the test suite before I merge this PR.

**Pull request merged**